app/api/job/createJob.ts

```diff
@@ -33,7 +32,12 @@ export async function createJobAppointment({
     // Calculate appointmentStartsBefore
     const appointmentStartsBefore = new Date(new Date(endTime).getTime() + 30 * 60000).toISOString();
 
-    logger.info("Checking if customer already exists");
+    logger.info({
+      message: "Checking if customer already exists",
+      customerName: name,
+      customerPhone: phone,
+      customerEmail: email,
+    });
```

Comment:
@jemini-yen I'm going to merge this PR soon - but afterward can you make the rest of these fixes?

The fix is: the `message` param should be removed and the message should just be the second argument of the method call.

Reply:
Also- I challenge you to use AI to do it. :)